matlab/Glacier2/greeter/Greeter.ice

```diff
@@ -2,6 +2,7 @@
 
 #pragma once
 
+["matlab:identifier:visitorcenter"]
```

Comment:
namespaces are usually lowercase in MATLAB. It's unfortunate we can't easily rename namespace Ice, IceStorm etc.